lego/apps/users/models.py

```diff
@@ -197,6 +199,10 @@ def memberships(self):
             user=self,
         )
 
+    @abakus_cached_property
```

Comment:
Because of some strange behavior in `@cached_property` while saving a model instance, @kotlarz improved it, and renamed it to `@abakus_cached_property`. I'm not 110% sure how they works.